examples/example-host-creation.py

```diff
@@ -42,11 +42,11 @@
 # Create host.
 data = {
     "common_name": "api-test",
-    "addresses": ["test.api.host.net", "123.123.123.123"],
+    "addresses": ["test.api.host.net", "127.0.0.1"],
     "services": [
         {
             "service": "SSH",
-            "address": "123.123.123.123",
+            "address": "127.0.0.1",
```

Comment:
Using loopback address can cause confusion. The proper fix is to use private network address (192.168.0.0/16, 10.0.0.0/8). The fixed address review could be accepted as "Safe" in Sonar. 

```
HOST_DNS = "test.privx.ssh.com"
HOST_IP = "192.168.0.10"
...
data = {
    "addresses": [HOST_DNS, HOST_IP],
    "services": [
        {
            "service": "SSH",
            "address": HOST_IP,
            ...
        }
    ]
}
```